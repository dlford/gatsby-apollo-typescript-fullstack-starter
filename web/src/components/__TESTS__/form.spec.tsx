import React from 'react'
import { render } from '@testing-library/react'

import Form from '../form'

describe('Form', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <Form>
        <label htmlFor='email'>Email Address</label>
        <input type='email' name='email' />
        <label htmlFor='password'>Password</label>
        <input type='password' name='password' />
        <button type='submit'>Submit</button>
      </Form>,
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
