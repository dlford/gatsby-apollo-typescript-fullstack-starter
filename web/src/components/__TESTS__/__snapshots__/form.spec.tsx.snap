// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form matches snapshot 1`] = `
<DocumentFragment>
  <form
    class="css-j0pxcv"
  >
    <label
      for="email"
    >
      Email Address
    </label>
    <input
      name="email"
      type="email"
    />
    <label
      for="password"
    >
      Password
    </label>
    <input
      name="password"
      type="password"
    />
    <button
      type="submit"
    >
      Submit
    </button>
  </form>
</DocumentFragment>
`;
